framework/src/main/java/org/checkerframework/framework/type/typeannotator/DefaultForTypeAnnotator.java

```diff
@@ -116,15 +119,16 @@ public void addTypeClass(
         }
     }
 
-    public void addTypeName(Class<?> typeName, AnnotationMirror theQual) {
-        String typeNameString = typeName.getCanonicalName();
-        boolean res = qualHierarchy.updateMappingToMutableSet(typeNames, typeNameString, theQual);
+    /** Add default qualifier, {@code theQual}, for the given class. */
+    public void addClasses(Class<?> clazz, AnnotationMirror theQual) {
```

Comment:
In a few places, in particular field and method names, I would change the `Classes` to `Types`, to be consistent with the name of the annotation attribute.